source/material_model/rheology/visco_plastic.cc

```diff
@@ -271,7 +271,7 @@ namespace aspect
             // one order or the other.
             if (friction_models.get_friction_mechanism() != static_friction)
               current_friction = friction_models.compute_friction_angle(current_edot_ii,
-                                                                        j, current_friction);
+                                                                        j, current_friction, in.position[i]);
```

Comment:
Please wrap this line to put each parameter on its own line. Also do we need the if-condition in line 272? The `compute_friction_angle` function correctly handles the case of static friction as far as I can tell.

Reply:
I removed the if now, as you are right, that this should not change the behaviour. I wasn't sure if it would make a difference in walltime and also which one would be the faster option.